modules/databricks-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/databricks/Databricks.scala

```diff
@@ -88,6 +88,13 @@ object Databricks {
               Fragment.const0(DatabricksEventsTable.statement(qualify(EventsTable.MainName)))
             )
 
+          /**
+           * Overrides the default by omitting the `DROP TABLE` step and instead use `CREATE OR
+           * REPLACE`
+           */
+          override def prepareAlertTable: List[Statement] =
+            List(Statement.CreateAlertingTempTable)
```

Comment:
SQL statement for CreateAlertingTempTable isn't updated in Databricks.scala. I guess it should be replaced with `CREATE OR REPLACE TABLE` statement. I will update it 👍